ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime.Azure/AzureClientExtensions.cs

```diff
@@ -123,14 +148,14 @@ public static class AzureClientExtensions
         }
 
         /// <summary>
-        /// Gets operation result for PUT and PATCH operations.
+        /// Gets operation result for PUT and PATCH operations. (Deprecated, please use GetLongRunningOperationResultAsync)
```

Comment:
Is this method summary correct? It seems like this operation is the new unified GetLongRunningOperationResult, not the deprecated GetPutOrPatchOperationResult